src/BasePlatform.js

```diff
@@ -79,10 +79,12 @@ export default class BasePlatform {
      * with getUserMedia, return a string explaining why not.
      * Otherwise, return null.
      */
-    screenCaptureErrorString() {
+    screenCaptureErrorString(): string {
         return "Not implemented";
     }
 
+    isElectron(): boolean { return false; }
```

Comment:
how do you mean? I can't do `PlatformPeg.get() instanceof ElectronPlatform` from the react-sdk side of things. How about a generic `getFlags()` method on BasePlatform and descendants that would have a prop of isElectron = true on Electron Platform, no idea what else it'd be useful for but it seems like the most generic way of solving this